packages/building-rollup/README.md

```diff
@@ -155,342 +168,209 @@ For example to add support for class properties:
 }
 ```
 
-## Extending the rollup config
+## Customizations
```

Comment:
Maybe we can 'go the extra mile' here and explain that the `createConfig` functions really just return a rollup configuration object that people can still manipulate when they need to add/customize stuff. Maybe it helps remove some of the 'magic'?

EDIT: nvm, you explain this further down below